File: Jacob Gichira Mbuco/Arrays and Strings/reverse_array.java
import java.util.*;
public class reverse_array {
     
   
    static void rvereseArray(
                    int start, int end)
    {
      
        int temp;
        int arr [] = {1,2,3,4,5};
          
        while (start < end)
        {
            temp = arr[start];
            arr[start] = arr[end];
            arr[end] = temp;
            start++;
            end--;
            
        }
      System.out.println(arr);
    }
}


       

File: Clare Limo/ArraysAndStrings/reverseArray.js
function reverseA(arr){
    let start = 0;
    let end = arr.length-1;

    while(start<end){
        let temp = arr[start];
        arr[start] = arr[end];
        arr[end] = temp;
        start++;
        end--;
    }

}
let array = [1,2,3,4,5];
reverseA(array);
console.log(array);